CONTRIBUTING.md

```diff
@@ -2,13 +2,9 @@
 
 Hi! Thank you for your interest in contributing to WooCommerce Services. We appreciate it.
 
-Although what you see today is a plugin, the goal for  WooCommerce Services is to be a fully integrated part of WooCommerce core. To get us from feature plugin to core, WooCommerce Services development is now in ALPHA - this means that you should expect frequent and sometimes large changes as features are added. During ALPHA testing, we'll be fixing problems and working to launch a feature-complete BETA. Successful BETA will culminate in a PR for inclusion into a future WooCommerce release.
+The emphasis for initial release of WooCommerce Services is shipping simplified. We are providing a Shipping Zones-compatible USPS and Canada Post shipping methods. Shipping Zones are [an exciting new feature of WooCommerce 2.6](https://woocommerce.wordpress.com/2016/02/10/shipping-zones-to-ship-with-2-6/). We are also allowing to print discounted USPS shipping labels.
```

Comment:
Is this paragraph useful? I don't think shipping simplified is our emphasis, and the explanation of shipping zones doesn't seem relevant any more since it's been around for a while.